recipes/spirv-tools/all/conanfile.py

```diff
@@ -89,35 +96,61 @@ def package(self):
-        # apparantly only exports the C-interface of the library.
-        # The test_package.c is used when testing the c_only option
-        # The C-interface is ALWAYS built as a shared-object as per the original
-        # CMakeLists.txt file
-        self.cpp_info.libs.append("SPIRV-Tools-reduce")
-        self.cpp_info.libs.append("SPIRV-Tools-link")
-        self.cpp_info.libs.append("SPIRV-Tools-opt")
-        self.cpp_info.libs.append("SPIRV-Tools")
-
+        self.cpp_info.names["pkg_config"] = "SPIRV-Tools"
+        # SPIRV-Tools
+        self.cpp_info.components["spirv-tools-core"].names["cmake_find_package"] = "SPIRV-Tools"
+        self.cpp_info.components["spirv-tools-core"].names["cmake_find_package_multi"] = "SPIRV-Tools"
+        self.cpp_info.components["spirv-tools-core"].libs = ["SPIRV-Tools"]
+        self.cpp_info.components["spirv-tools-core"].requires = ["spirv-headers::spirv-headers"]
+        if self.settings.os == "Linux":
+            self.cpp_info.components["spirv-tools-core"].system_libs.append("rt")
+        if not self.options.shared and self._stdcpp_library:
+            self.cpp_info.components["spirv-tools-core"].system_libs.append(self._stdcpp_library)
+        # SPIRV-Tools-opt
+        self.cpp_info.components["spirv-tools-opt"].names["cmake_find_package"] = "SPIRV-Tools-opt"
+        self.cpp_info.components["spirv-tools-opt"].names["cmake_find_package_multi"] = "SPIRV-Tools-opt"
+        self.cpp_info.components["spirv-tools-opt"].libs = ["SPIRV-Tools-opt"]
+        self.cpp_info.components["spirv-tools-opt"].requires = ["spirv-tools-core", "spirv-headers::spirv-headers"]
         if self.settings.os == "Linux":
-            self.cpp_info.system_libs.extend(["m", "rt"]) # m for SPIRV-Tools-opt, rt for SPIRV-Tools
+            self.cpp_info.components["spirv-tools-opt"].system_libs.append("m")
+        # SPIRV-Tools-link
+        self.cpp_info.components["spirv-tools-link"].names["cmake_find_package"] = "SPIRV-Tools-link"
+        self.cpp_info.components["spirv-tools-link"].names["cmake_find_package_multi"] = "SPIRV-Tools-link"
+        self.cpp_info.components["spirv-tools-link"].libs = ["SPIRV-Tools-link"]
+        self.cpp_info.components["spirv-tools-link"].requires = ["spirv-tools-core", "spirv-tools-opt"]
+        # SPIRV-Tools-reduce
+        self.cpp_info.components["spirv-tools-reduce"].names["cmake_find_package"] = "SPIRV-Tools-reduce"
+        self.cpp_info.components["spirv-tools-reduce"].names["cmake_find_package_multi"] = "SPIRV-Tools-reduce"
+        self.cpp_info.components["spirv-tools-reduce"].libs = ["SPIRV-Tools-reduce"]
+        self.cpp_info.components["spirv-tools-reduce"].requires = ["spirv-tools-core", "spirv-tools-opt"]
 
         bin_path = os.path.join(self.package_folder, "bin")
         self.output.info('Appending PATH environment variable: %s' % bin_path)
         self.env_info.path.append(bin_path)
+
+    @property
+    def _stdcpp_library(self):
```

Comment:
I think we already added conan helper for that: https://docs.conan.io/en/latest/reference/tools.html?highlight=cppstd#tools-stdcpp-library

Reply:
Yes, but CI has not been updated yet: #1918

Reply:
Now it's updated: https://github.com/conan-io/conan-center-index/wiki/Deployment-Changelog#24-june-2020---1055-cest